certify-service/src/main/resources/application-local.properties

```diff
@@ -241,7 +241,7 @@ mosip.certify.integration.scan-base-package=io.mosip.certify.mock.integration
 mosip.certify.integration.audit-plugin=LoggerAuditService
 mosip.certify.integration.vci-plugin=MockVCIssuancePlugin
 mosip.certify.mock.vciplugin.verification-method=${mosip.certify.authn.jwk-set-uri}
-
+mosip.certify.mock.authenticator.get-identity-url=http://localhost:9000/path/to/server
```

Comment:
why is this new config added, I don't see any place within this PR we are using it.

Reply:
I have missed these changes from the cherry-pick, I'll add it again.

Reply:
> why is this new config added, I don't see any place within this PR we are using it.

Without this configuration, the mock-plugin wasn't running out of the box, hence I added this configuration, should I move it under the plugin specific comment or remove this config alltogether?

Reply:
still dint get why is this required ?